AssetManagementTestProject/TestCases/US304ChangePasswordTest.cs

```diff
@@ -0,0 +1,22 @@
+﻿using AssetManagementTestProject.TestSetup;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
```

Comment:
Remove unused code for all files changed

Reply:
Done